cypress-io/cypress
# fix: various test failures due to dependencies not being flagged as `norewrite`

- Closes #22992

### User facing changelog


n/a

### Additional details


The main changes here were to force several dependencies into a `norewrite` status. They each had issues that were causing problems when the snapshot doctor would rewrite the file. Forcing them into a `norewrite` status fixes the problem. Long term, I logged a follow up issue to see if some of these problems could be discovered automatically.

### Steps to test


No real functionality change here. To test, on `feature/v8-snapshots` you can uncomment the tests that this PR comment out and run them and see that they fail. Then, see that they pass on this PR

### How has the user experience changed?


n/a

### PR Tasks


- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
